### DISCUSSION TITLE
[@types/node] dns.setDefaultResultOrder() not implemented

### DISCUSSION THREAD

**Discussion by BenSjoberg** [DefinitelyTyped/DefinitelyTyped#55904] [Issues with a @types package] [answered]
This function was added in Node 16.4.0: https://nodejs.org/dist/latest-v16.x/docs/api/dns.html#dns_dns_setdefaultresultorder_order

It allows you to globally define the default value of the "verbatim" option in dns.lookup.

I've never contributed to DefinitelyTyped before, but I could probably come up with a PR for this, since it's a very simple function.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
> I've never contributed to DefinitelyTyped before, but I could probably come up with a PR for this, since it's a very simple function.
- no worries, just do it
- add method declaration, it shoudl accept string unions (allowed params)
- add a simple tests for those
- method should have docs, see other methods in this specific module for pointers,
txh!

**Comment by westy92** [comment, marked as answer]
I have added [a PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57176) to fix this.

**Reply by westy92** [reply]
This has been merged and released. This discussion should now be resolved!

**Reply by BenSjoberg** [reply]
Done. Sorry it took me so long to get to it. :)
